tasks/actorstate/task.go

```diff
@@ -61,8 +61,8 @@ func (t *Task) ProcessActors(ctx context.Context, ts *types.TipSet, pts *types.T
 	log.Debugw("processing actor state changes", "height", ts.Height(), "parent_height", pts.Height())
 
 	report := &visormodel.ProcessingReport{
-		Height:    int64(ts.Height()),
-		StateRoot: ts.ParentState().String(),
+		Height:    int64(pts.Height()),
+		StateRoot: pts.ParentState().String(),
```

Comment:
Just noting that:
- block processor doesn't need this change since it only acts on a single tipset.
- message processor already uses parent tipset.
- message execution processor uses parent tipset.